api/inc/svc_exports.h

```diff
@@ -112,6 +112,8 @@
 #define UVISOR_SVC_ID_BOX_NAMESPACE_FROM_ID UVISOR_SVC_CUSTOM_TABLE(17)
 #define UVISOR_SVC_ID_DEBUG_REBOOT          UVISOR_SVC_CUSTOM_TABLE(18)
 #define UVISOR_SVC_ID_DEBUG_REGISTER_BOX    UVISOR_SVC_CUSTOM_TABLE(19)
+#define UVISOR_SVC_ID_PAGE_MALLOC           UVISOR_SVC_CUSTOM_TABLE(20)
```

Comment:
Please keep an eye on https://github.com/ARMmbed/uvisor/pull/221 in case it gets merged first (it adds these IDs as well).